### PR TITLE
Feature conversation

### PR DESCRIPTION
**Pull request by mixcas**
Add conversation feature

### CODE REVIEW

File: css/site.styl [hunk 1]
```diff
@@ -107,13 +114,29 @@ img
...
-input
-  height: 0
-  width: 0
-  opacity: 0
-  text-transform: uppercase
 
+// CONVERSATION FORM
+.conversation
+
+  #custom-form
```
Comment (varffs) [inline css/site.styl:121]: how can an ID be nested under a class? There is only ever 1 `#custom-form` but more than one `.conversation`?

File: js/main.js [hunk 2]
```diff
@@ -118,6 +152,45 @@ Adventure = {
...
+  chat: function(conversation) {
+    var _this = this;
+
+    // Get the conversation
+    conversation = typeof conversation !== 'undefined' ? _this.story[conversation].options : _this.currentPlace.options;
+
+    var chatForm = '<form id="custom-form">';
+
+    // Generate radio buttons 
+    for(var ffs = 1; ffs <= _.size(conversation); ffs++) {
```
Comment (varffs) [inline js/main.js:164]: ffs @mixcas formatting

File: js/main.js [hunk 3]
```diff
@@ -118,6 +152,45 @@ Adventure = {
...
+    _this.$customForm.html(chatForm);
+    _this.$customForm = $('#custom-form-container');
+    _this.say(_this.currentPlace.description);
+    _this.$mainContainer.addClass('conversation');
+
+    // Set new focus element
+    _this.$focus = $('#radio-1');
+
+    // Bind submit
+    _this.$customForm.children().bind('submit', function(event) {
```
Comment (varffs) [inline js/main.js:183]: why just a `.children()` not a more accurate selector?

File: css/site.styl (hunk 1 above)
Comment (varffs) [inline css/site.styl:121]: I see that this conversation class is a class for the main container. But what need is there to nest like this why not just use the `#custom-form` id?

**Comment by mikeyrayvon** [comment]
if these are comments are not regarding bugs lets leave them until later.

**Comment by varffs** [comment]
@mikeyrayvon :( you already merged without bugs so thats fine but @mixcas needs to see them and think about them cus that how code review needs to be even if we are in a hurry

File: css/site.styl (hunk 1 above)
Comment (mixcas) [inline css/site.styl:121]: cuz that class change… i actually really dislike using id's for styling but i dont wanna go there now

Comment (varffs) [inline css/site.styl:121]: point being do u need to qualify that id with the class? Cus I can't see why?

Comment (mixcas) [inline css/site.styl:121]: duh yes

File: js/main.js (hunk 2 above)
Comment (mixcas) [inline js/main.js:164]: what? the for? sorry

**Pull request merged**